src/App.css

```diff
@@ -1,17 +1,30 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@500&display=swap');
+
 .App {
   text-align: center;
 }
 
+.upper {
+  text-transform: uppercase;
+}
+
 .nav {
   width: 100%;
   display: flex;
   justify-content: left;
   padding: 2rem;
   background-color: #fff;
+  align-items: center;
 }
```

Comment:
Great job so far. It would be better if you decrease your `navbar` height. You can add  `height` property and you can use the value from `Zeplin` website. kindly fix it.
Your design
![image](https://github.com/AlselwiAisha/Bookstore/assets/101523582/6a6d57ba-1a55-4dc8-9644-d8bbfe4103d4)

The given design
![image](https://github.com/AlselwiAisha/Bookstore/assets/101523582/9f2b55a7-e62d-47d5-884e-91d4522afdd5)